dashboard/src/app/workspaces/workspace-details/workspace-overview/workspace-details-overview.controller.ts

```diff
@@ -121,6 +127,10 @@ export class WorkspaceDetailsOverviewController {
       deRegistrationFn();
     });
 
+    this.cheDashboardConfigurationService.ready.then(() => {
+      this.enabledKubernetesNamespaceSelector = this.cheDashboardConfigurationService.enabledFeature('kubernetesNamespaceSelector');
```

Comment:
Does it make sense to move `kubernetesNamespaceSelector` somewhere in constants file?

Reply:
Method `enabledFeature()` accepts the [type alias](https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases) which is union of two strings in this case - `kubernetesNamespaceSelector` and `workspaceSharing`, so the autocompletion works well.
But I'm totally OK to use there enum instead.

Reply:
Ah, union )
Code makes impression that it's just string.
It's up to you whether union or enum more suitable here.